tools/java/org/unicode/cldr/util/DtdData.java

```diff
@@ -654,20 +653,47 @@ public void handleEndDtd() {
     /**
      * Normal version of DtdData
      * Note that it always gets the trunk version
+     * @deprecated depends on static configuration (CLDR_DIR)
      */
     public static DtdData getInstance(DtdType type) {
-        return CACHE.get(type);
+        return getInstance(type, CLDRConfig.getInstance().getCldrBaseDirectory());
     }
-
+    
     /**
      * Special form using version, used only by tests, etc.
      */
     public static DtdData getInstance(DtdType type, String version) {
-        DtdData simpleHandler = new DtdData(type, version);
-        XMLFileReader xfr = new XMLFileReader().setHandler(simpleHandler);
         File directory = version == null ? CLDRConfig.getInstance().getCldrBaseDirectory()
             : new File(CLDRPaths.ARCHIVE_DIRECTORY + "/cldr-" + version);
 
+        return getInstance(type, version, directory);
+    }
+    
+    private static final Map<Pair<DtdType, File>, DtdData> OTHERCACHE = new  HashMap<>();
```

Comment:
It's a bit odd for this to be called "other cache" when the "original" cache field no longer exists. Perhaps just rename it back to "CACHE" or similar.

Reply:
I wasn't sure if I needed to bring back the 'other cache' for performance reasons. A quick check shows that running all of the console checks didn't degrade much with this current code, even better with a concurrent map. I'll rename it.

Reply:
^ PTAL